packages/styles/scss/fonts/README.md

```diff
@@ -0,0 +1,134 @@
+Carbon:
+
+| Font                      | Available | Entrypoint                    |
+| ------------------------- | --------- | ----------------------------- |
+| IBM Plex Mono             | ✅        | `scss/fonts/mono`             |
+| IBM Plex Sans Arabic      | ✅        | `scss/fonts/sans-arabic`      |
+| IBM Plex Sans Devanagari  | ✅        | `scss/fonts/sans-devanagari`  |
+| IBM Plex Sans Hebrew      | ✅        | `scss/fonts/sans-hebrew`      |
+| IBM Plex Sans KR          |           |                               |
+| IBM Plex Sans JP          |           |                               |
+| IBM Plex Sans Thai Looped | ✅        | `scss/fonts/sans-thai-looped` |
+| IBM Plex Sans Thai        | ✅        | `scss/fonts/sans-thai`        |
+| IBM Plex Sans             | ✅        | `scss/fonts/sans`             |
+| IBM Plex Serif            | ✅        | `scss/fonts/serif`            |
+
+By default, Carbon provides the default font weights for: IBM Plex Mono, IBM
+Plex Sans, and IBM Plex Serif. To bring in additional fonts, you can include the
+`@carbon/styles/scss/fonts` entrypoint and configure it:
+
+```scss
+@use '@carbon/styles/scss/fonts' with (
+  $fonts: (
+    IBM-Plex-Sans-Arabic: true,
+  ),
+);
+```
+
+You can also configure it to disable specific fonts:
+
+```scss
+@use '@carbon/styles/scss/fonts' with (
+  $fonts: (
+    IBM-Plex-Sans: false,
+  ),
+);
+```
+
+If you would like to disable all fonts, you can use the `$css--font-face` flag
+in:
+
+```scss
+@use '@carbon/styles/scss/config' with (
+  $css-font-face: false,
+);
+```
+
+## Features
+
+Each font is available as an entrypoint in the `fonts` folder. You can use these
+entrypoints to include specific font weights, styles, and more for IBM Plex. For
+example, if you only want to include the regular font weight for IBM Plex Sans
+then you could do the following:
+
+```scss
+@use '@carbon/styles/scss/fonts/sans';
+
+@include sans.regular($weights: normal);
+```
+
+### Custom font src resolver
+
+By default, `@carbon/styles/scss/fonts` attempts to resolve fonts directly from
+the `@ibm/plex` package. You can configure the location of these font files in
+two ways:
+
+- Use the `$font-path` option to define a path "prefix"
+- Provide a custom resolver to point to where your font files are hosted
+
+The `$font-path` options is available in `scss/config`:
+
+```scss
+@use '@carbon/styles/scss/config' with (
+  $font-path: 'https://cdn.custom-font-path.com/fonts',
+);
+```
+
+And allows you to change the beginning part of each `src` URL that is generated.
+You can also completely override the default behavior and provide your own URL
+resolver through `scss/fonts/src`:
+
+```scss
+@use 'sass:meta';
+@use './my-custom-resolver' as resolver;
+@use '@carbon/styles/scss/fonts/src' with (
+  // Here, "resolve" is the name of the function in the "resolver" module
+  $resolver: meta.get-function('resolve', 'resolver')
+);
```

Comment:
It's awesome that you were able to figure this out. I see below that you use `meta.call()` to call it, very cool! TIL higher order functions are possible in sass 🎉